Felienne/hedy
# [FIX] Returns unsaved changes warning on switching levels

**Description**
In this PR we fix the unsaved changes warning when switching levels. It was already there, but was no longer working due to the removal of the `auth.profile` value. It now works again. We should note that due to the removal of the `auth.profile` value the prompt will always be shown as the front-end no longer "knows" if there is a user logged in -> this functionality is all handled by the back-end.

**Fixes**
This PR fixes #2313

**How to test**
Make sure you are logged in, create a program in any level and use the "go to next level" button. Verify that you receive a "are you sure" prompt.
